src/citrine/resources/process_template.py

```diff
@@ -98,9 +98,9 @@ def _build_child_objects(cls, data: dict, session: Session = None):
 
         Returns
         -------
-        dict
-            The serialized material template, but its conditions are now a list of object pairs
-            of the form [ConditionTemplate, Bounds], and the parameters are
+        None
+            The serialized material template is modified so that its conditions are now a list
```

Comment:
```suggestion
            The serialized process template is modified so that its conditions are now a list
```
Again, seeming type mismatch